src/reducers/Actions.js

```diff
@@ -0,0 +1,16 @@
+export const Actions = {
+  ADD_ITEM_TO_CART: "ADD_ITEM_TO_CART",
+  ADD_OR_REMOVE_ITEM_TO_WISHLIST: "ADD_OR_REMOVE_ITEM_TO_WISHLIST",
+  INITIALIZE_CART: "INITIALIZE_CART",
+  INITIALIZE_WISH: "INITIALIZE_WISH",
+  INITIALIZE_PRODUCTS: "INITIALIZE_PRODUCTS",
+  UPDATE_QUANTITY: "UPDATE_QUANTITY",
+  REMOVE_FROM_CART: "REMOVE_FROM_CART",
+  PRICE_HIGH_TO_LOW: "PRICE_HIGH_TO_LOW",
+  PRICE_LOW_TO_HIGH: "PRICE_LOW_TO_HIGH",
+  FILTER_OUT_OF_STOCK: "FILTER_OUT_OF_STOCK",
+  FILTER_FAST_DELIVERY: "FILTER_FAST_DELIVERY",
+  CLEAR_ALL_FILTERS: "CLEAR_ALL_FILTERS",
+  UPDATE_SEARCH_TEXT: "UPDATE_SEARCH_TEXT",
+  OPEN_OR_CLOSE_TOAST: "OPEN_OR_CLOSE_TOAST"
+};
```

Comment:
Liked this. Separated all the constants into one file. Looks good!
It helps to keep track as well. Though keeping camel case naming in mind, good idea to export these as `actions` rather than `Actions`